pages/goals/[slug].tsx

```diff
@@ -129,15 +129,16 @@ const EditGoalsPage = () => {
             <input
               type="date"
               className="w-1/2 p-2 text-left border rounded-md shadow-md md:w-1/4 text-murkrow"
-              value={format(
-                new Date(editedGoalValues.targetDate),
-                "yyyy-MM-dd"
-              )}
+              value=""
```

Comment:
having an empty string here means the date will never actually update: 

<img width="416" alt="Screen Shot 2022-12-19 at 3 08 00 PM" src="https://user-images.githubusercontent.com/98791180/208545476-e1c75e71-6ba3-44bf-bbcf-340456d4427a.png">

Reply:
great catch! that was done by accident.. put the following code back in :

`value={format(
                new Date(editedGoalValues.targetDate),
                "yyyy-MM-dd"
              )}`